app/assets/javascripts/controllers/middleware_server/middleware_server_controller.js

```diff
@@ -120,7 +120,14 @@ function MwServerControllerFactory($scope, miqService, isGroupDeployment) {
 
   $scope.resetJdbcDriverForm = function () {
     $scope.jdbcDriverModel = {};
-    $scope.jdbcDriverModel.serverId = angular.element('#server_id').val();
+    if (angular.element('#server_id').val()) {
```

Comment:
I'd suggest storing the `angular.element('#server_id').val()` to some variable so you don't do twice DOM access, but only once.